app/api/event_copyright.py

```diff
@@ -40,22 +40,33 @@ class EventCopyrightList(ResourceList):
 
     def query(self, view_kwargs):
```

Comment:
You can put all those as well in this PR if you want. And your pr seems to coincide to some extent with #3861 . Check that and maybe remove it? Your choice though how you want to do.